modules/lunamediaBidAdapter.js

```diff
@@ -210,8 +211,18 @@ function createVideoRequestData(bid, bidderRequest) {
   let topLocation = getTopWindowLocation(bidderRequest);
   let topReferrer = getTopWindowReferrer();
 
-  let sizes = getVideoSizes(bid);
-  let firstSize = getFirstSize(sizes);
+  // if size is explicitly given via adapter params
+  var paramSize = getVideoBidParam(bid, 'size');
```

Comment:
Why is `var` being used in the code in this PR when `let` and `const` are used throughout the existing code?

Reply:
updated code.